package.json

```diff
@@ -8,14 +8,16 @@
   },
   "scripts": {
     "build": "webpack --mode production",
-    "start": "npm run build && node src/server/index.js",
+    "start": "node src/server/index.js",
```

Comment:
We pull out the build step to be it's own thing. Devs will need to be mindful to build before starting.